keylime/payloads/controller_payload/local_action_node_failover.py

```diff
@@ -0,0 +1,15 @@
+import os
+import asyncio
+
+async def execute(revocation):
+    """Node fail over script
+
+    This script will be sent to the OpenShift controller along with a payload/
+    folder.
+    When a node is compromised. Keylime verifier will inform all machines
+    running an agent to peform location actions. The local action in this case
```

Comment:
"location actions" seems like a typo, should that be "local actions" or something else?

Reply:
yup :) , will fix that.